source/shared/cpp/ObjectModel/CaptionSource.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "CaptionSource.h"
+#include "ParseUtil.h"
+
+using namespace AdaptiveCards;
+
+CaptionSource::CaptionSource()
+{
+}
+
+Json::Value CaptionSource::SerializeToJsonValue() const
```

Comment:
I'm wondering if you need propagate change to iOS and Android

Reply:
I believe we just need to do this for UWP and JS, @jonmill can correct me here if I'm wrong